server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/mailbox/Quotas.java

```diff
@@ -22,9 +22,9 @@
 import java.util.Objects;
 import java.util.Optional;
 
-import org.apache.james.core.quota.QuotaCount;
-import org.apache.james.core.quota.QuotaSize;
-import org.apache.james.core.quota.QuotaValue;
+import org.apache.james.core.quota.QuotaCountLimit;
+import org.apache.james.core.quota.QuotaLimitValue;
+import org.apache.james.core.quota.QuotaSizeLimit;
```

Comment:
I find it suspicious that QuotaCountValue is not present in the imports. Is everything allright?

Reply:
I fixed the type parameters of `Quotas.Value` as they are not really used it doesn't change a lot of things. But its more correct in a domain sens now. Thanks.